static/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -153,7 +153,17 @@ const SidebarDropdown = ({api, org, orientation, collapsed, config, user}: Props
                   </React.Fragment>
                 )}
               </DemoModeGate>
-            </OrgAndUserMenu>
+
+              <Divider />
+
+              <React.Fragment>
+                <div>
```

Comment:
I don't think we even need this div?

Reply:
Confirmed. Works fine without this and `React.Element`. 👍

Reply:
Done in 7357660782d533efe67b34de17be806cefe25a81.